nengo_loihi/loihi_cx.py

```diff
@@ -421,15 +421,24 @@ def format(self, **kwargs):
         self.synapse_fmt.set(**kwargs)
 
 
+class Spike(object):
+    __slots__ = ['axon_id', 'atom']
+
+    def __init__(self, axon_id, atom=0):
```

Comment:
Could you add a docstring to this? Mostly I'm curious what an `atom` is.

Reply:
Yes, I'll do that.